test-utils/src/dev/test_cmds.rs

```diff
@@ -37,13 +37,17 @@ pub fn path_to_executable(cmd_name: &str) -> PathBuf {
 }
 
 #[track_caller]
-pub fn assert_exit_code(exit_status: ExitStatus, code: u32) {
+pub fn assert_exit_code(
+    exit_status: ExitStatus,
+    code: u32,
+    stderr_text: impl AsRef<str>,
```

Comment:
Nit: I would just use `&str` as the type here. I don't think we need this level of genericism here since all the call sites seem to just be doing `&foo` where `foo` is a `String`.